src/content/eth2/roadmap/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Eth2 roadmap
+description: The path to Eth2 explained
+lang: en
+sidebar: true
+---
+
+# Eth2 roadmap
+
+<Subtitle>The road to Eth2 is long and winding but there are some important checkpoints along the way. </Subtitle>
+
+## Where we are
+
+Currently, teams are building out Eth2 clients and in the process of testing them. A client is a piece of software that serves as a "node" in the Ethereum network, helping to read and validate transactions.
+
+There are many different types of clients because Ethereum encourages a multi-client world. As Danny Ryan puts it, "Ethereum is the abstract set of rules that makes up the protocol rather than any specific implementation of those set of rules".
+
+And they're being tested across different testnets so that teams can iron out any bugs.
+
+[More info on teams building Eth2](https://docs.ethhub.io/ethereum-roadmap/ethereum-2.0/eth2.0-teams/teams-building-eth2.0/) _– ETHHub_
+
+Testing is public so you can check it out here:
+
+- [eth2stats](https://eth2stats.io/onyx-testnet)
+- [beaconcha.in](https://beaconcha.in/)
+- [Etherscan](https://beaconscan.com/)
+
+### Join testing {#join-testing}
+
+If you want to play a part in Eth2, help test the different clients.
+
+<Button to="/eth2/get-involved#go-bug-hunting">Start testing</Button>
+
+## Phase 0: the beacon chain
+
+Expected: 2020
+
+In Eth2 there will be lots of parallel chains (or [shards](/en/eth2/shard-chains/) with their own validators processing transactions.
+
+<InfoBanner emoji=":thinking_face:">
+    <b>What's a validator?</b> An actor who proposes and confirms the state of new blocks on the network.
+ </InfoBanner>
+
+Shards are where the additional scalability and throughput will come from. However all the shards will need consensus for the network to remain safe.
+
+**The beacon chain is a new blockchain at the core of Ethereum that will provides consensus**
+
+The beacon chain at first will be responsible for registering validator [stakes](/en/eth2/proof-of-stake/). However, at first a smart contract on mainnet will actually accept the stake.
```

Comment:
This paragraph is a bit confusing... both sentences mention "at first". There's also not much explanation of the "smart contract". I suggest expanding on it or taking it out if we don't think it's critical knowledge..